src/convert_book/options.rs

```diff
@@ -1,9 +1,9 @@
 pub const RELEASE_DATE: &'static str = "2016-10-01";
```

Comment:
```suggestion
 pub const RELEASE_DATE: &'static str = "2019-05-04";
```
Change the release date